crates/rpc/src/lib.rs

```diff
@@ -18,6 +20,31 @@ impl BeerusRpc {
     }
 
     pub async fn run(self) -> Result<(SocketAddr, ServerHandle), Error> {
+        let local_spec_version =
+            self.spec_version().await.expect("Should never happen");
```

Comment:
> Should never happen

If I had a nickel every time something that should never happen actually happens... I would have had a few nickels :) Why not add an error context and fail-fast (with `?`)?

One of the ways to proceed might be extracting the version as a constant and adding a unit-test for `spec_version()` to return a value that matches the constant. Then there is no need for a local call to `spec_version` and simple check against constant is enough.

Reply:
Hey, I'm the one who suggested that!

You're right though, a constant would avoid this.

Reply:
I am not pushing for a constant, it is only "one of [many] ways to proceed".

It is `expect` that looks unusual. But I guess failing to return a static string should indeed never happen.